nix/README.md

```diff
@@ -66,7 +66,7 @@ In its simplest form, the `nix copy` command can be used to stash a specific pac
 
 By using inputDerivation, we can create a meta-package that contains all the packages in our devShell.
 
-As an exmample, this copy will stash the s2n-tls devShell:
+As an example, this copy will stash the s2n-tls devShell:
```

Comment:
now my typos PR is less relevant

Reply:
No still very relevant 🙄 have you seen the rest of the library?